src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -134,16 +149,43 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           // We export the longest valid prefix
           String exportedNamespace = nameUtil.findLongestNamePrefix(lhs, symbols.keySet());
           if (exportedNamespace != null) {
-            convertExportAssignment(child, exportedNamespace, symbols.get(exportedNamespace));
+            convertExportAssignment(
+                child, exportedNamespace, symbols.get(exportedNamespace), filename);
             // Registers symbol for rewriting local uses
-            registerLocalSymbol(child.getSourceFileName(), exportedNamespace, exportedNamespace,
+            registerLocalSymbol(
+                child.getSourceFileName(),
+                exportedNamespace,
+                exportedNamespace,
                 symbols.get(exportedNamespace));
           }
           break;
         default:
           break;
       }
     }
+
+    private void directlyExportSymbol(Node namedNode, Node parent, String filename) {
+      if (!fileToModule.containsKey(filename)) {
+        return;
+      }
+      FileModule module = fileToModule.get(filename);
+      Node child = namedNode.getFirstChild();
+      String nodeName = checkNotNull(child.getQualifiedName());
+
+      ExportedSymbol exportedSymbol = ExportedSymbol.of(filename, nodeName, nodeName);
+      if (module.exportedNamespacesToSymbols.containsValue(nodeName)) {
```

Comment:
It was actually `export const A = B;` but I looked into this more with a fresh head and was able to refactor the code, so we don't need to do the AST transformation here, and, instead, can consolidate it in one place.